lib/storage/data/external/utils.js

```diff
@@ -45,7 +45,7 @@ const utils = {
         const metaObj = obj || {};
         Object.keys(metaObj).forEach(key => {
             const newKey = key.substring(11);
-            newObj[newKey] = metaObj[key];
+            newObj[newKey] = `${metaObj[key]}`;
```

Comment:
If we need to convert a string, best to make it explicit:
```suggestion
            newObj[newKey] = string(metaObj[key]);
```